Sources/ZippyJSON/ZippyJSONDecoder.swift

```diff
@@ -37,19 +38,15 @@ func isOnSimulator() -> Bool {
   #endif
 }
 
-public final class ZippyJSONDecoder {
+public final class ZippyJSONDecoder: Sendable {
```

Comment:
This compiles without having to resort to `@unchecked Sendable`, so we get a compile-time guarantee of thread-safety :)

Reply:
very nice!